Sofa/framework/Helper/src/sofa/helper/OptionsGroup.cpp

```diff
@@ -31,7 +31,6 @@ class OptionsGroup;
 //////////////////////////////////////////////////////////////////////////////////////////////////////
 OptionsGroup::OptionsGroup() : textItems()
 {
-    selectedItem=0;
```

Comment:
This is certainly breaking. Even if not set here, `selectedItem` will have a default value. Might as well control it no ? Or add an internal mechanism that track if the optionGroup has been set at least once, to know if we can trust this attribute.

Reply:
I agree. `selectedItem` will have a value if you define it or not. I don't see how it solves your problem. Could you imagine a fix in the `CollisionResponse` class instead?